### PR TITLE
Update service button to have the correct path

### PR DESCRIPTION
**Pull request by summer-cook**
# Summary
Updates the button on the service page to have the correct path so that it will show the dynamic form / the new request form of that service

# Related
#120 
webstore pr: https://github.com/scientist-softserv/webstore/pull/122

# Video
https://share.getcloudapp.com/9ZuzKZyd

### CODE REVIEW

File: src/compounds/ItemPage/ItemPage.jsx [hunk 1]
```diff
@@ -15,8 +16,11 @@ const ItemPage = ({ title, titleStyle, description, descriptionStyle, img }) =>
       <Row className='mt-3'>
         <Col xs={12} md={6} className='d-flex flex-column'>
           <TextBox text={description} style={descriptionStyle} size='medium' />
-          {/* TODO(summer-cook): Update this button's href so it is pointing at the item's form instead of blank request form */}
-          <Button href='/requests/new' className='align-self-start mt-4'>Initiate Request</Button>
+          <LinkedButton
+            addClass='align-self-start mt-4'
+            buttonProps={{label: 'Initiate Request'}}
```
Comment (alishaevn) [inline src/compounds/ItemPage/ItemPage.jsx:21]:
```suggestion
            buttonProps={{ label: 'Initiate Request' }}
```

File: src/compounds/ItemPage/ItemPage.jsx [hunk 2]
```diff
@@ -29,7 +33,9 @@ const ItemPage = ({ title, titleStyle, description, descriptionStyle, img }) =>
 ItemPage.propTypes = {
   description: PropTypes.string.isRequired,
   descriptionStyle: PropTypes.shape({}),
-  img: Image.defaultProps.isRequired,
+  id: PropTypes.number.isRequired,
+  img: PropTypes.shape({}),
```
Comment (alishaevn) [inline src/compounds/ItemPage/ItemPage.jsx:37]: why change the `img` propType definition?

**Review by alishaevn — CHANGES_REQUESTED** [review]

**Comment by alishaevn** [comment]
holding approval until we resolve [this conversation](https://github.com/scientist-softserv/webstore/pull/122/files#r1073702358) in the webstore related pr.

File: src/compounds/ItemPage/ItemPage.jsx (hunk 2 above)
Comment (summer-cook) [reply src/compounds/ItemPage/ItemPage.jsx:37]: it had an error in the console with how it previously was

Comment (alishaevn) [reply src/compounds/ItemPage/ItemPage.jsx:37]: ah. weird. can you list the src and alt props inside the shape though?

Comment (summer-cook) [reply src/compounds/ItemPage/ItemPage.jsx]: done!

File: src/compounds/ItemPage/ItemPage.jsx [hunk 3]
```diff
@@ -1,37 +1,49 @@
...
+          <TextBox text={description || snippet} style={descriptionStyle} size='medium' />
+          <LinkedButton
+            addClass='align-self-start mt-4'
+            buttonProps={{ label: 'Initiate Request' }}
+            path={{ pathname: `/requests/new/${slug}`, query: { id } }}
+          />
         </Col>
         <Col xs={12} md={6} className='mt-4 mt-md-0 me-md-auto d-flex justify-content-start justify-content-md-end'>
-          <Image {...img} />
+          <Image {...img} addClass='img-fluid'/>
```
Comment (alishaevn) [inline src/compounds/ItemPage/ItemPage.jsx:26]:
```suggestion
          <Image {...img} addClass='img-fluid' />
```

File: src/compounds/ItemPage/ItemPage.stories.jsx [hunk 4]
```diff
@@ -11,25 +11,36 @@ const Template = (args) => <ItemPage {...args} />
...
   img: {
     alt: 'test tubes',
     src: item,
   },
-  title: 'CRISPR[DEMO]',
   titleStyle: {},
+  ware: {
+    // eslint-disable-next-line max-len
+    description: 'Using advanced CRISPR technology, add or delete base pairs at a specifically targeted DNA loci. Using advanced CRISPR technology, add or delete base pairs at a specifically targeted DNA loci. Using advanced CRISPR technology, add or delete base pairs at a specifically targeted DNA loci. Using advanced CRISPR technology, add or delete base pairs at a specifically targeted DNA loci. Using advanced CRISPR technology, add or delete base pairs at a specifically targeted DNA loci. Using advanced CRISPR technology, add or delete base pairs at a specifically targeted DNA loci.',
+    id: '4356',
```
Comment (alishaevn) [inline src/compounds/ItemPage/ItemPage.stories.jsx:23]:
```suggestion
    id: 4356,
```

File: src/compounds/ItemPage/ItemPage.stories.jsx [hunk 5]
```diff
@@ -11,25 +11,36 @@ const Template = (args) => <ItemPage {...args} />
...
   img: {
     alt: 'test tubes',
     src: item,
     width: 650,
   },
-  title: 'CRISPR[DEMO]',
   titleStyle: {},
+  ware: {
+    description: '',
+    id: '4356',
```
Comment (alishaevn) [inline src/compounds/ItemPage/ItemPage.stories.jsx:40]:
```suggestion
    id: 4356,
```

**Review by alishaevn — CHANGES_REQUESTED** [review]

**Review by alishaevn — APPROVED** [review]

**Pull request merged**